packages/components/src/templates/drawer.stories.ts

```diff
@@ -52,31 +53,116 @@ export const Default = {
+              <sd-navigation-item slot="children" indented class="pl-8">Insurance and Guarantees</sd-navigation-item>
+            </sd-navigation-item>
+            <sd-navigation-item vertical indented slot="children">
+              Distinction
+              <sd-navigation-item slot="children" indented class="pl-8">Investor Education Center</sd-navigation-item>
+              <sd-navigation-item slot="children" indented class="pl-8">Risk Management Tools</sd-navigation-item>
+              <sd-navigation-item slot="children" indented class="pl-8">Insurance and Guarantees</sd-navigation-item>
+            </sd-navigation-item>
+            <sd-navigation-item vertical indented slot="children">
+              Our Management
+              <sd-navigation-item slot="children" indented class="pl-8">Investor Education Center</sd-navigation-item>
+              <sd-navigation-item slot="children" indented class="pl-8">Risk Management Tools</sd-navigation-item>
+              <sd-navigation-item slot="children" indented class="pl-8">Insurance and Guarantees</sd-navigation-item>
+            </sd-navigation-item>
+          </sd-navigation-item>
+          <sd-navigation-item vertical
+            >Union Investment Group
+            <sd-navigation-item vertical indented slot="children" current>
+              Investor protection
+              <sd-navigation-item slot="children" indented class="pl-8">Investor Education Center</sd-navigation-item>
+              <sd-navigation-item slot="children" indented class="pl-8">Risk Management Tools</sd-navigation-item>
+              <sd-navigation-item slot="children" indented class="pl-8">Insurance and Guarantees</sd-navigation-item>
+            </sd-navigation-item>
+            <sd-navigation-item vertical indented slot="children">
+              Distinction
+              <sd-navigation-item slot="children" indented class="pl-8">Investor Education Center</sd-navigation-item>
+              <sd-navigation-item slot="children" indented class="pl-8">Risk Management Tools</sd-navigation-item>
+              <sd-navigation-item slot="children" indented class="pl-8">Insurance and Guarantees</sd-navigation-item>
+            </sd-navigation-item>
+            <sd-navigation-item vertical indented slot="children">
+              Our Management
+              <sd-navigation-item slot="children" indented class="pl-8">Investor Education Center</sd-navigation-item>
+              <sd-navigation-item slot="children" indented class="pl-8">Risk Management Tools</sd-navigation-item>
+              <sd-navigation-item slot="children" indented class="pl-8">Insurance and Guarantees</sd-navigation-item>
+            </sd-navigation-item>
+          </sd-navigation-item>
+        </nav>
+      </div>
     </sd-drawer>
+    <script>
+      function handleSecondLevel() {
+        document.querySelector('.level-one').classList.toggle('hidden');
```

Comment:
Please choose one naming convention for consistency

Reply:
Updated